README.md

```diff
@@ -5,14 +5,22 @@
 SparkR is an R package that provides a light-weight frontend to use Spark from
 R.
 
+*NOTE: As of April 2015, SparkR has been [merged](https://github.com/apache/spark/pull/5096) into Apache Spark and is shipping in an upcoming release (1.4) due early summer 2015. This repo currently targets users using released versions of Spark.*
+
```

Comment:
Lets add a note here saying new contributions should be opened against the Spark repository with a link to https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark

Also the JIRA link below, can you update it to say that bugs should be reported against Spark JIRA ?